app/app/src/main/kotlin/com/hedvig/android/app/navigation/HedvigNavHost.kt

```diff
@@ -326,9 +342,12 @@ internal fun HedvigNavHost(
     helpCenterGraph(
       hedvigDeepLinkContainer = hedvigDeepLinkContainer,
       navigator = navigator,
-      onNavigateToQuickLink = { backStackEntry, quickLinkDestination ->
+      onNavigateToQuickLink = onNavigateToQuickLink@{ backStackEntry, quickLinkDestination ->
         val destination: Destination = when (quickLinkDestination) {
-          QuickLinkChangeAddress -> ChangeAddress
+          QuickLinkChangeAddress -> {
+            navigateToMovingFlow()
+            return@onNavigateToQuickLink
```

Comment:
this is a bit new for me :)

Reply:
Haha yeah I know it's a bit special. it's so we can short-circuit out of the lambda, despite the fact that we are trying to do an assignment to `val destination: Destination = when...`
I don't necessarily love this too much, but I know it's temporary while the feature flag exists, so I thought it's okay.